src/index.html

```diff
@@ -1,13 +1,313 @@
+
+            <p class="content__text">
+              The app senses when you're nearby
+               to unlock automatically. GPS tracking
+               so you know where your bike is and can track
+               it anytime.
+            </p>
+          </div>
+
+          <div class="content__explained content__range">
+            <div class="content__flex">
+              <img
+               src="/img/range--1.png"
+               alt="Range and Integrated"
+               class="range--1__pic">
+
+              <img
+               src="/img/range--2.png"
+               alt="Range and Integrated brakes"
+               class="range--2__pic">
+            </div>
+
+            <h2 class="content__header">
+              Range & Integrated lights
+            </h2>
+
+            <p class="content__text">
+              The removable battery has up to 70km battery
+              autonomy and weighs only 2.4 kg. Lights integrated
+              into the frame give you always-on
+              visibility day and night.
+            </p>
+          </div>
+
+          <div class="content__explained content__brakes">
+            <div class="content__flex">
+              <img
+              src="/img/brakes--1.png"
+              alt="brakes"
+              class="brakes--1__pic">
+
+              <img
+              src="/img/brakes--2.png"
+              alt="brakes"
+              class="brakes--2__pic">
+
+            </div>
+
+            <h2 class="content__header">
+              Hydraulic disc brakes & Lightweight
+            </h2>
+
+            <p class="content__text">
+              Brakes with total stopping power the second
+              you make contact. The removable battery has
+              up to 70km battery autonomy and weighs only
+              2.4 kg. Lights integrated into the frame give
+              you always-on visibility day and night.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <div class="button__conten">
+        <div class="button__cover">
+          <button class="button">
+            <p class="button__text">
+              Explore
+            </p>
+          </button>
+        </div>
+      </div>
+
+      <section class="section section4" id="contacts">
+        <h2 class="section__header section__header4">
+          Contact us
+        </h2>
+
+        <div class="contact__content">
+          <div class="contact--us">
+            <div class="form">
+              <input
+              type="text"
+              name="name"
+              class="contact__input"
+              placeholder="Name">
+
+              <input
+              type="text"
+              name="email"
+              class="contact__input"
+              placeholder="Email">
+
+              <textarea
+              cols="1"
+              rows="1"
+              placeholder="Message"
+              class="contact__input">
+              </textarea>
```

Comment:
```suggestion
              <textarea
              cols="1"
              rows="1"
              placeholder="Message"
              class="contact__input"
            ></textarea>
```

It should help with extra spaces in textarea field